libraries/apollo-api/src/wasmJsMain/kotlin/com/apollographql/apollo3/api/apolloUnsafeCast.wasmJs.kt

```diff
@@ -0,0 +1,5 @@
+package com.apollographql.apollo3.api
+
+actual inline fun <reified T> Any.apolloUnsafeCast(): T {
+  TODO()
```

Comment:
`@JsExport` doesn't seem to be much of a thing yet, something to investigate, see https://slack-chats.kotlinlang.org/t/16102339/does-jsexport-work-in-wasm-if-yes-could-the-jsany-bound-of-u#04a7fdd4-dabc-4578-82bb-a2e748c704ac